guides/doc-Upgrading_and_Updating/topics/con_upgrading-prerequisites.adoc

```diff
@@ -0,0 +1,28 @@
+[[upgrading_prerequisites]]
+== Prerequisites
+
+Upgrading to {Project} {ProjectVersion} affects your entire {Project} infrastructure.
+Before proceeding, complete the following:
+
+* Read the {ProjectName} {ProjectVersion} {ReleaseNotesURL}[Release Notes].
+* Plan your upgrade path.
+For more information, see xref:upgrade_paths[].
+* Plan for the required downtime. {Project} services are shut down during the upgrade.
+The upgrade process duration might vary depending on your hardware configuration, network speed, and the amount of data that is stored on the server.
++
+Upgrading {Project} takes approximately 1{range}2 hours.
++
+Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
+
+* Ensure that you have sufficient storage space on your server.
+For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartPr
+oxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+* Back up your {ProjectServer} and all {SmartProxyServers}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+* Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
```

Comment:
I'm wondering if we have some guide noting the changes between versions. Otherwise this isn't really actionable for users.